rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -31,6 +31,7 @@ import io.swagger.annotations.ApiModelProperty
 @CompileStatic
 class BakeRequest {
 
+  String request_id = UUID.randomUUID().toString()
```

Comment:
Please add a comment here indicating that this arg is not expected to be passed in from the outside. Or if you consider it a reasonable mechanism for passing an external correlation id, describe that usage a bit.

Feels like this is intended for internal usage. If that is so, let's document it in a comment.